Create ui-c8y-1021-34-0-service-management-actions-in-device-tab-services-list.md (#2838)
Cumulocity-IoT/c8y-docs: content/change-logs/device-management/ui-c8y-1021-34-0-service-management-actions-in-device-tab-services-list.md

---
date: ""

> title: Service management actions in device tab services list.

Reviewer: ```suggestion
title: Service management actions added to every service in device tab services list.
```

> title: Service management actions added to every service in device tab services list.

Reviewer 2: ```suggestion
title: Service management actions added to the services list in the device details
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4311
version: 1021.34.0
---

> Service management actions in device tab services list.

Reviewer: Service management actions have been added directly to the services list in the device details tab. With this change, users can now conveniently perform service-related actions such as starting, stopping, or restarting a service right from the device services list.

> Service management actions in device tab services list.

Reviewer: ```suggestion
Service management actions have been added directly to the services list in the device details tab. With this change, users can now conveniently perform service-related actions such as starting, stopping, or restarting a service right from the device services list.
```

> Service management actions have been added directly to the services list in the device details. With this change, users can now conveniently perform service-related actions such as starting, stopping, or restarting a service right from the device services list.

Reviewer 2: ```suggestion
Service management actions have been added to each service in the services list in the device details. With this change, users can now conveniently perform service-related actions such as starting, stopping, or restarting a service right from the device services list.
```

> Service management actions have been added directly to the services list in the device details tab. With this change, users can now conveniently perform service-related actions such as starting, stopping, or restarting a service right from the device services list.

Reviewer 2: ```suggestion
Service management actions have been added directly to the services list in the device details. With this change, users can now conveniently perform service-related actions such as starting, stopping, or restarting a service right from the device services list.
```
